opm/output/eclipse/RestartIO.cpp

```diff
@@ -767,6 +967,77 @@ namespace {
         ::Opm::OpmLog::info(msg);
     }
 
+
+    const std::vector<int>  writeGlobalRestart( int report_step, int sim_step, double seconds_elapsed,
+                                                const Schedule& schedule, const EclipseGrid& grid, const EclipseState& es,
+                                                const Action::State& action_state,  const WellTestState& wtest_state,
+                                                const SummaryState& sumState, const UDQState& udqState, bool ecl_compatible_rst,
+                                                bool write_double, EclIO::OutputStream::Restart& rstFile, const std::vector<RestartValue>& values,
+                                                std::optional<Helpers::AggregateAquiferData>& aquiferData)
+    {
+        // Write GLOBAL RESTART
```

Comment:
This comment serves no purpose as it just repeats the function name.  Please either just remove it or add some more information.